python/lsst/daf/butler/core/dataUnit.py

```diff
@@ -541,14 +510,22 @@ def _initDataUnitJoins(self, config, builder):
                     isView = "sql" in tableDescription
             lhs = frozenset((dataUnitJoinDescription.get('lhs', None)))
             rhs = frozenset((dataUnitJoinDescription.get('rhs', None)))
+            dataUnitNames = lhs | rhs
+            relates = frozenset(self[name] for name in dataUnitNames)
             summarizes = dataUnitJoinDescription.get('summarizes', None)
+            spatial = dataUnitJoinDescription.get("spatial", False)
```

Comment:
Inconsistent mix of single and double quotes for string literals all over :slightly_smiling_face: PEP8 says "Pick a rule and stick to it", I guess our rule is "whatever goes".

Reply:
I prefer double quotes everywhere and mostly use that. But I don't speak for everyone so perhaps we should do a poll for `daf_butler`...